card/lib/card/query.rb

```diff
@@ -61,7 +61,8 @@ class Query
                            link_to linked_to_by
                            include included_by                                ),
       conjunction:     %w(and or all any),
-      special:         %w(found_by not sort match complete extension_type),
+      special:         %w(found_by not sort match name_match complete junction_complete
```

Comment:
re `name_match`, my memory was that you could do:
```
"match":"X" # meaning name or content
"name":["match", "X"] # meaning name only, or
"content":["match", "X"] # meaning content only.
```
Does that not work?  Or is this different?